### ISSUE TITLE
Getting to the other vertex after outE() in MATCH

### ISSUE THREAD

**Issue by obi1kenobi** [orientechnologies/orientdb#7802]
## Expected behavior  
Presumably, it should be possible to start a MATCH query at a vertex `A`, traverse to the attached edge via, e.g., an `outE()` command, and then get to the vertex `B` on the other side of that edge. For example:
```
MATCH {
    class: V,
    as: start_vertex
}.outE() {
    as: edge_in_between
}.inV() {
    as: end_vertex
} RETURN $matches
```

## Actual behavior  
It does not seem that `inV()` or `outV()` are supported in MATCH. Furthermore, using regular `in()/out()` after an edge-fetching query does not seem to work -- queries like the following return 0 results:
```
MATCH {
    class: V,
    as: start_vertex
}.outE() {
    as: edge_in_between
}.in() {
    as: end_vertex
} RETURN $matches
```

I haven't been able to find a way to do this correctly, either through experimentation or by reading through the docs. The [MATCH query docs](https://orientdb.com/docs/2.2/SQL-Match.html) explicitly state that `outE()` and `inE()` are supported, but say nothing about the corresponding `outV()` or `inV()` functions that would get us back from edge-land to vertex-land.

## Steps to reproduce  
Try a query with the above structure on any data set. I've been using the example databases that come with OrientDB.

**Comment by luigidellaquila** [comment]
Hi @obi1kenobi 

It's pretty strange, we have test cases for these things that pass correctly.
I'm checking it now

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @obi1kenobi 

I tried this query on GratefulDeadConcerts, MovieRatings and OpenBeer databases, with latest 2.2.29

```
MATCH {
    class: V,
    as: start_vertex
}.outE() {
    as: edge_in_between
}.inV() {
    as: end_vertex
} RETURN $matches
```

and it seems to work fine. Any other idea on how to reproduce the problem?

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Oh, please check your second query, you have `in()` instead of `inV()`...

Thanks

Luigi

**Comment by obi1kenobi** [comment]
Right, I was saying that I found `in()` to not work after `inV()` also didn't work. However -- you're right, I rebuilt my docker image and reinitialized the database, and `inV()` does work now. Maybe I somehow got my database in a corrupted state, sorry for the false alarm.

You might want to update the docs to mention that `inV()` and `outV()` are supported though. They currently don't mention that, and it might be difficult for a new user to figure out that's what they need to do.
